en/contribute/code.md

```diff
@@ -20,6 +20,12 @@ All code contributions have to be under the permissive [BSD 3-clause license](ht
 
 PX4 uses the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html), with the following (minimal) modifications:
 
+:::note
+Not all PX4 source code matches the style guide, but any _new code_ that you write should do so — in both new and existing files.
+There is no requirement to update other code in changed files to match the guide.
+However if you do, we recommend that you use `clang-format` and add formatting changes as a separate commit.
```

Comment:
Also, there's technically a `clang-tidy` target also. I don't know if either are really supported. I'll look into it a bit more this weekend :).

Reply:
Great. I'm thinking here that until we have commitment to add the tidy we don't need to mention it. The formatter is pretty much a no-brainer, so it is worth mentioning here.

Reply:
Ah! Sorry @hamishwillee, but we don't have a `clang-format` integrated just yet (it's inside Matthias's PR at the moment). There's Astyle targets (check or fixing the formatting), which would be `make check_format` for check and for fix it would be `make format`. I think it's done on every PR automatically at the moment. Although I think it is doing it not completely at least, or it's missing some formatting.

We can still say it, and change it if we decide to go with clang-format in the future :). What do you think?

Reply:
I think we'll just keep quiet about this now - I'll delete the line.
However if you notice that the PR to change formatting goes in, please let me know and we'll update (or just do a PR!)

Reply:
```suggestion
```